src/rt/sched/behaviourcore.h

```diff
@@ -746,15 +752,17 @@ namespace verona::rt
           cown_to_behaviour_slot_map.get() + cown_count,
           compare);
 
-      // First phase - Prepare phase
-      // For each unique chain you need to keep the cown, the first index,
-      // the Slot for that cown in the last body in the chain
-      // the transfer count for reference counting, and a bool that will be
-      // used later to indicate that the chain had no predecessor
+      // Helper struct to be used after building the chains in the next phases
+      struct ChainInfo {
+        Cown *cown;
+        size_t first_body_index;
+        Slot *slot;
```

Comment:
I think this is the last slot in the chain?
```suggestion
        Slot *last_slot;
```